src/components/HorizontalCard/HorizontalCard.jsx

```diff
@@ -1,33 +1,44 @@
+import { useCartContext } from "../../context/cart-context";
 import "./HorizontalCard.css";
 
-const HorizontalCard = () => {
+const HorizontalCard = ({ item }) => {
+  const { cartState, cartDispatch } = useCartContext();
```

Comment:
const {title , description} = item

Reply:
is used my useCartContext there so I cant change put there what you said it will gave error